doc/man7/ossl-guide-migration.pod

```diff
@@ -157,10 +157,6 @@ will still work. However, their applicability will be limited.
 
 New algorithms provided via engines will still work.
 
-Engine-backed keys can be loaded via custom B<OSSL_STORE> implementation.
```

Comment:
The migration guide should stay intact; this should not get backported and will be replaced for 4.0 anyway.

Reply:
yes, it was, without touching migration guide, I get
```
doc/man7/ossl-guide-migration.pod:1: reference to non-existing ENGINE_load_private_key(3)
```
Not sure how to fix it without removal of that paragraph. Just remove the symbol mention?